docs/usage/bot-comparison.md

```diff
@@ -18,7 +18,7 @@ If you see anything wrong on this page, please let us know by creating a [Discus
 | Supported languages                       | [List for Dependabot](https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/about-dependabot-version-updates#supported-repositories-and-ecosystems) | [List for Renovate](https://docs.renovatebot.com/modules/manager/)                                       |
 | Customizability                           | Low                                                                                                                                                                          | High                                                                                                     |
 | Show changelogs                           | Yes                                                                                                                                                                          | Yes                                                                                                      |
-| Compatibility score badges                | 1                                                                                                                                                                            | 4                                                                                                        |
+| Compatibility score badges                | One badge with overall compatibility score                                                                                                                                   | Four badges showing: Age, Adoption, Passing, Confidence                                                  |
```

Comment:
Would it be possible to have hyperlinks to each of these badges explaining what they mean?